tools/wpt-report-builder/builder.mjs

```diff
@@ -61,19 +61,36 @@ function readReport(filePath) {
 }
 
 const jsonPath = process.argv[2];
-const outputPath = process.argv[3];
-const filteredOutputPath = process.argv[4];
+const jsonInteropPath = process.argv[3];
```

Comment:
Optional:
Would it be possible to make the script API more generic, like this: `node tools/wpt-report-builder/builder.mjs --report-path {outputPath} --filter-2023 {true | false} {list-of-json-reports}`? The important part is providing several report paths. This would allow generating report in sharded WPT run.

Reply:
Can you open an issue about this, this would require some refactoring, and most likely using `yargs` to handle the argument parts.

Reply:
sure: https://github.com/GoogleChromeLabs/chromium-bidi/issues/2240